server/server_test.go

```diff
@@ -1556,38 +1556,39 @@ func (cli *testServerClient) runTestLoadData(t *testing.T, server *Server) {
 		dbt.MustExec("drop table if exists pn")
 	})
 
+	// TODO: disabled
 	// Test with upper case variables.
-	cli.runTestsOnNewDB(t, func(config *mysql.Config) {
```

Comment:
@hawkingrei PTAL at this failed case

Reply:
https://github.com/pingcap/tidb/issues/41596

Reply:
LGTM